clang/docs/checkedc/3C/CONTRIBUTING.md

```diff
@@ -67,6 +67,31 @@ you can use it in scripts), and display some information about any
 failures, which may or may not be enough for you to understand what
 went wrong.
 
+For deeper troubleshooting, run the following in your build directory
+to build all dependencies of the test suite:
+
+```
+ninja check-3c-deps
+```
+
+Then run the following in the `clang/test/3C` directory:
+
+```
+llvm-lit -vv TEST.c
+```
+
+where `TEST.c` is the path of the test you want to run (you can also
+specify more than one test). This assumes you've put the `bin`
+subdirectory of your build directory on your `$PATH` or arranged some
+other means of running `llvm-lit` from there. The first `-v` makes
+`llvm-lit` display the stdout and stderr of failed tests; the second
+makes it display the `RUN` commands as they execute so you can tell
+which one failed.
+
+Every `.c` file under `clang/test/3C` is a test file. There are a few
+in subdirectories, so `*.c` will not pick up all of them; instead you
+can use `llvm-lit -vv .`.
```

Comment:
The two periods may be confusing when rendered. Drop the second one (or rearrange the sentence so they are not both in a row).

Reply:
Fixed by inserting another piece of information that is helpful anyway.